x-pack/legacy/plugins/maps/server/maps_telemetry/telemetry_task.js

```diff
@@ -41,35 +41,28 @@ export function registerMapsTelemetryTask(server) {
       title: 'Maps telemetry fetch task',
       type: TELEMETRY_TASK_TYPE,
       timeout: '1m',
-      numWorkers: 2,
       createTaskRunner: telemetryTaskRunner(server),
     },
   });
 }
 
 export function telemetryTaskRunner(server) {
-
   return ({ taskInstance }) => {
     const { state } = taskInstance;
     const prevState = state;
 
-    const callCluster = server.plugins.elasticsearch.getCluster('admin')
-      .callWithInternalUser;
+    const callCluster = server.plugins.elasticsearch.getCluster('admin').callWithInternalUser;
```

Comment:
I'm not 100% sure about this, but these small cleanup changes might get switched back to the former when this file gets converted to Typescript and `prettier` is enforced, so it might not be worth it to have these changes in the commit. I leave it up to you

Reply:
Sigh, yeah, that's the autoformatting that Prettier has been configured to do.
I assumed it's better to go with the default that's configured, otherwise it'll just reformat for the next person.

Reply:
Oh wait, you're saying Prettier isn't switched on for js, only ts?
I wonder why it's auto formatting then... seems weird.
I'll double check, might be a global thing, in which case, I'll revert those reformats.